admin_guide/backup_restore.adoc

```diff
@@ -511,6 +512,101 @@ member d266df286a41a8a4 is healthy
 xref:bringing-openshift-services-back-online[Bringing {product-title}
 Services Back Online].
 
+[[cluster-restore-etcd-quorum]]
+== Restoring etcd quorum
+
+If you lose etcd quorum, you must take down your etcd cluster and form a new 
+one. You can use one healthy etcd node to form a new cluster, but you must
+remove all other healthy nodes.
+
+[NOTE]
+====
+During etcd quorum loss, applications that run on {product-title} are
+unaffected. However, the the platform functionality is limited to read-only
+operations. You cannot take action such as scaling an application up or down, 
+changing deployments, or running or modifying builds.
+====
+
+. To confirm the loss of etcd quorum, run the following command and confirm
+that the cluster is unhealthy:
++
+----
+# etcdctl2 cluster-health
+member 165201190bf7f217 is unhealthy: got unhealthy result from https://master-0.example.com:2379
+member b50b8a0acab2fa71 is unreachable: [https://master-1.example.com:2379] are all unreachable
+member d40307cbca7bc2df is unreachable: [https://master-2.example.com:2379] are all unreachable
+cluster is unhealthy
+----
++
+Note the member IDs and host names of the hosts. You use one of the nodes that
+can be reached to form a new cluster.
+
+[[cluster-restore-etcd-quorum-backup-etcd]]
+=== Backing up etcd
+include::day_two_guide/topics/etcd_backup.adoc[]
+
+=== Removing failed etcd hosts
+include::day_two_guide/topics/removing_etcd_host.adoc[]
+
+[[cluster-restore-etcd-quorum-single-node]]
+=== Creating a single-node etcd cluster
+
+Make a remaining etcd node a standalone etcd cluster. By taking this step, you 
+restore the full functionality of your {product-title} instance.
+
+[[cluster-restore-etcd-quorum-single-node-v2-v3]]
+==== Using v2 and v3 data to create an etcd cluster
```

Comment:
Would this be better as a standalone module?

Reply:
Maybe. I don't know where else it would be used, though, so I don't know if it's module-worthy. It depends on how diligently we say that modules must be worth reusing.